app/templates/search.html

```diff
@@ -1,14 +1,39 @@
 {% extends "base.html" %}
 {% import '_snippets.html' as snippets %}
+{% block scripts %}
+  <script>
```

Comment:
@subhankarb you should not need JS to do that. You can just set it in the form using the query parameter passed into the page ...